libexec/mcd/mcd-duty

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+### mcd-duty -- Calculate a system duty
+###
+### https://docs.makerdao.com/smart-contract-modules/rates-module
+###
+### Usage: mcd duty [<percent>]
+###
+### example:
+###
+### $ mcd duty 4
+### > 1000000001243680656318820312
+set -e
+
+require-percent() {
+  [ -n "$1" ] || mcd --fail "mcd-duty: Please specify a percentage (i.e. 4.25 == 4.25%)"
```

Comment:
Seems like this check fits better in `require-numeric` (or maybe the regex could even catch the empty string edge case).